shpkpr/deployment/standard.py

```diff
@@ -18,8 +23,15 @@ def __init__(self, marathon_client, timeout, app_definitions, **kw):
     def execute(self, force=False):
         """Execute standard Marathon deployment.
         """
+        app_ids = ", ".join([a["id"] for a in self.app_definitions])
+        logger.info("Executing standard deployment: {0}".format(app_ids))
+
         deployment = self.marathon_client.deploy(
             self.app_definitions,
             force=force,
         )
-        return deployment.wait(timeout=self.timeout)
+
+        logger.info("Waiting for marathon deployment to complete: {0}".format(deployment.deployment_id))
```

Comment:
nab: some stutter here, potentially makes sense to rename to `deployment.id`

Reply:
`id` is a [built-in](https://docs.python.org/3/library/functions.html#id) in Python and I'd rather not shadow it, hence the stutter. Open to other suggestions though.